backend/apps/accounts/views.py

```
@@ -107,36 +113,41 @@ class SignupView(generics.CreateAPIView):
-    ]
+    ],
 )
 def register(request):
-    # ... view logic
+    pass
+
 
 @extend_schema(
     summary="Login user",
     description="Authenticate user and return JWT token",
     request=UserLoginSchema,
```

Ruff (0.15.20)

[error] Undefined name `UserLoginSchema`

(F821)